docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2776,17 +2918,149 @@ The `ingester_client` block configures how the distributors connect to the inges
 ```yaml
 # Configures the gRPC client used to communicate with ingesters from
 # distributors, queriers and rulers.
-# The CLI flags prefix for this block configuration is: ingester.client
-[grpc_client_config: <grpc_client>]
+grpc_client_config:
+  # (advanced) gRPC client max receive message size (bytes).
```

Comment:
Hum why did this change? I also don't see the removal of `<grpc_client>` section, so is it repeated now?

Reply:
Oh, I know why, it's because you wrapped the config with another type. If you remove the wrapping (see the other comment about just moving `RegisteredFlags` to dskit) this will be reverted.